windowsforms/graphics/README.md

```diff
@@ -1,6 +1,6 @@
 # Windows Forms Graphics Demo
 
-This Windows Forms sample was ported to .NET Core from [a previous .NET Framework 4.5 sample](https://code.msdn.microsoft.com/windowsdesktop/CSWinFormGraphics-4572a5f0).
+This Windows Forms sample was ported to .NET Core from a previous .NET Framework 4.5 sample.
```

Comment:
Should this point to https://github.com/dotnet/samples/tree/master/windowsforms/graphics?